packages/cli/__tests__/zosfiles/__system__/create/ds/cli.files.create.ds.system.test.ts

```diff
@@ -39,6 +41,8 @@ describe("Create Data Set", () => {
         REAL_SESSION = TestEnvironment.createZosmfSession(TEST_ENVIRONMENT);
 
         user = defaultSystem.zosmf.user.trim().toUpperCase();
+        pass = defaultSystem.zosmf.password.trim().toUpperCase();
```

Comment:
Will explicitly making password and host uppercase cause an unintended failure?

Reply:
Thanks for catching, I've updated this section to remove `toUpperCase()`